lib/diagnostic.js

```diff
@@ -37,7 +40,7 @@ let scopes = [/* your response here */];
 /*1 */ alderaan = { name: 'Alderaan' };
 /*2 */ deathStar = {
 /*3 */   fire: function fire(){
-/*4 */     return this.name;
+/*4 */     return console.log(this.name);
```

Comment:
You weren't supposed to change this function. Additionally, `console.log` returns `undefined`, so it doesn't make sense to try and pass that out.